src/SWP/Bundle/FacebookInstantArticlesBundle/SWPFacebookInstantArticlesBundle.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Facebook Instant Articles Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Bundle\FacebookInstantArticlesBundle;
+
+use SWP\Bundle\StorageBundle\DependencyInjection\Bundle\Bundle;
+use SWP\Bundle\StorageBundle\Drivers;
+
+class SWPFacebookInstantArticlesBundle extends Bundle
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getSupportedDrivers()
+    {
+        return [
+            Drivers::DRIVER_DOCTRINE_ORM,
+        ];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getModelClassNamespace()
+    {
+        return 'SWP\Bundle\FacebookInstantArticlesBundle\Model';
```

Comment:
would be nice to create a component for this bundle and move some classes that dont use bundle integration, there